src/plone/restapi/tests/test_resolveuid.py

```diff
@@ -23,7 +23,15 @@ def setUp(self):
             )
         ]
         self.doc2 = self.portal[
-            self.portal.invokeFactory("Document", id="doc2", title="Target Document")
+            self.portal.invokeFactory(
```

Comment:
@cekk just wondering why this happens here. Should black prevent changes like this in PRs? I mean if black auto-formats things this isn't something that should change in any way...

Reply:
i think that this is my problem. I set black to auto-format on save in my editor, but i forgot to disable it when i was working on this pr.
Should i revert these changes?

Reply:
auto-formatting in the ide is fine and recommended. do you have flake8 auto-formatting enabled with a line-length limit? we set line-length to 200 in flake8 in the volto config because black takes care of that as well. if your ide black/flake8 picks up the custom config of the volto project things should be fine. maybe we should have a pre-commit hook that runs black to prevent something like this to happen on the CI...

Reply:
i'm not working on volto project for this pr, so probably that's the main problem.
And yes, i have a limit set to 80 because of old habits.
If you want i can re-format this file